### ISSUE TITLE
docker-arg not working

### ISSUE THREAD

**Issue by mg98** [serverless/serverless#8587]
Locally invoking a **go1.x** function with docker args does not work for me.

**Command**
```
sls invoke local -f createWeekReport --docker --docker-arg "-p 5432:5432"
```
or
```
sls invoke local -f createWeekReport --docker-arg "-p 5432:5432"
```

**Output**

```
Serverless: Running "serverless" installed locally (in service node_modules)
Serverless: Deprecation warning: Detected ".env" files. Note that Framework now supports loading variables from those files when "useDotenv: true" is set (and that will be the default from next major release)
            More Info: https://www.serverless.com/framework/docs/deprecations/#LOAD_VARIABLES_FROM_ENV_FILES
Serverless:  
Serverless: Learn more about configuration validation here: http://slss.io/configuration-validation
Serverless:  
 
  Serverless Error ---------------------------------------
 
  The file you provided does not exist: /Users/me/Projects/this-project/ 5432:5432
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              14.3.0
     Framework Version:         2.15.0 (local)
     Plugin Version:            4.2.0
     SDK Version:               2.3.2
     Components Version:        3.4.3
```

**Comment by medikoo** [comment]
@mg98 thanks for report. It appears it's a regression introduced with https://github.com/serverless/serverless/pull/7187

and clear issue with [`yargs-parser`](https://github.com/yargs/yargs-parser). I've reported the issue over there: https://github.com/yargs/yargs-parser/issues/344 Let's see what would be the response

**Comment by mg98** [comment]
Ok thanks! Knowing that, I figured I could work around this with `--docker-arg="-p 5432:5432"` 👍

**Comment by medikoo** [comment]
@mg98 I belive that was fixed with #8927
